web/crux/src/config/pino.logger.config.ts

```diff
@@ -0,0 +1,33 @@
+import pino from 'pino'
+
+const pinoLoggerConfig = {
+  pinoHttp: {
+    logger: pino({
+      customProps: () => ({
+        context: 'HTTP',
+      }),
+      level: process.env.LOG_LEVEL,
+      // Added this for further finetuning
+      // redact: {
+      //   paths: ['req.headers', 'remoteAddress'],
+      //   remove: true,
+      // },
+      transport:
+        // Use pino-pretty only in the development environment
+        process.env.NODE_ENV !== 'production'
+          ? {
+              target: 'pino-pretty',
+              options: {
+                singleLine: true,
+                ignore: 'pid,hostname',
+                translateTime: 'mm/dd/yyyy h:MM:ss',
+                messageFormat: '[{req.headers.host}{req.url}][{context}] {msg} [{responseTime} ms][{res.statusCode}]',
```

Comment:
In the long run, we should do this a better way.

Reply:
Can we include the `req.method` as well?

Reply:
Not now, we have to figure out the right format for these, but this is not the time 🙉